db/migrations/state/0006.sql

```diff
@@ -1,15 +1,28 @@
 -- +migrate Down
+UPDATE state.proof
+SET prover = prover_id;
+ALTER TABLE state.proof
+DROP COLUMN IF EXISTS prover_id;
 ALTER TABLE state.proof
 DROP COLUMN IF EXISTS created_at;
 ALTER TABLE state.proof
 DROP COLUMN IF EXISTS updated_at;
 ALTER TABLE state.proof
-DROP COLUMN IF EXISTS generating_since;
-ALTER TABLE state.proof
 ADD COLUMN  generating BOOLEAN DEFAULT FALSE;
+UPDATE state.proof
+SET generating = TRUE
+WHERE generating_since IS NOT NULL;
+ALTER TABLE state.proof
+DROP COLUMN IF EXISTS generating_since;
 
 -- +migrate Up
 ALTER TABLE state.proof
+ADD COLUMN prover_id VARCHAR;
+UPDATE state.proof
+SET prover_id = prover;
+UPDATE state.proof
+SET prover = NULL;
+ALTER TABLE state.proof
 DROP COLUMN IF EXISTS generating;
```

Comment:
To be consistent with the "migrate down" steps we can add an UPDATE query to update the field generating_since to NOW() when generating = TRUE

Reply:
@agnusmor, good point. I updated the migration file to apply your suggestion. I did this in #1603 because the code is related to that PR: ec4a096562dff1f26ef4a43ed71be7db54efe3f3.
I also addressed the `Migrate down` step to take care of setting the `generating` field to `true` when the `generating_since` was not null.